### DISCUSSION TITLE
Are `?raw` imports supported?

### DISCUSSION THREAD

**Discussion by soetz** [nuxt/nuxt#19694] [Questions]
Vite [allows importing an asset as a string using the `?raw` suffix](https://vitejs.dev/guide/assets.html#importing-asset-as-string). When I try to do this with Nuxt, as far as I can tell it works as expected, however it’s not part of the default type definition provided by Nuxt.

[Daniel mentioned here that all of Vite types may not be supported](https://github.com/nuxt/nuxt/pull/19338#discussion_r1123178681), is it the case with raw imports?

Also if I can give a hand with #14131 I’d be happy to.

Thanks a lot!

**Comment by mathix420** [comment]
I have exactly this requirement for a project where I want to import markdown files as raw text.

```ts
import prompt from '~/prompts/join.md?raw'
```

But I got the following issue: 
```
Error: Could not load [...]/join.md?raw (imported by transformers/schema/prompt.ts):
ENOENT: no such file or directory, open '[...]/join.md?raw'
```
